src/components/ctas/DemoCTA.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export default function DemoCTA() {
```

Comment:
Nit: Could we rename this to `BookDemoCTA`? That way we are explicit and it is clear for someone new to the codebase what the component would be used for.  Perhaps even `CTAButtonBookDemo`. Then we can call the other one `CTAButtonSignup`.  I start both with `CTAButton` as a means to have them all grouped together in the filesystem. A little nicety to make it easy to find all the CTA buttons in the sidebar :) (If you do not use CMD+P or you are not in VSCode or similar).  What do you think?

Reply:
Do you want me to push a commit to this branch or rename it once we merge?

Reply:
@schalkneethling You can push the commit here

Reply:
Actually makes sense to do it on a new PR after this is merged.